src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -98,7 +98,6 @@ func doMain() error {
 
 	// Schedule cleanup actions
 	defer logging.Close()
-	defer pacOff()
```

Comment:
It's me introduced the bug of not turning off pac by moving this line to below to near the call to `pacOn`. We can move it back here, or wrap the defer in a func, but I think add `exitFuncs` solely for this purpose is not worthy. How do you think @myleshorton ?